xbmc/dbwrappers/qry_dat.cpp

```diff
@@ -533,7 +533,7 @@ double field_value::get_asDouble() const {
 int64_t field_value::get_asInt64() const {
     switch (field_type) {
     case ft_String: {
-      return std::stoll(str_value);
+      return atoll(str_value.c_str());
```

Comment:
`std::atoll` here for consistency

Reply:
Changed. It's not consistent though because all other conversions are using the C functions (atoi, atof instead of std::atoi and std::atof) but I don't really mind either way.

Reply:
Using std is consistent with the rest of the clean-up and way new parts of Kodi will be implemented. Changing as we go does mean that things can get ahead of others.

Reply:
Understood, thanks for the review and for merging!